src/redux/__tests__/videoSlice.test.ts

```diff
@@ -133,8 +133,8 @@ describe('Video reducer', () => {
 
     // Assert
     const rootState = { videoState: nextState };
-    expect(selectCurrentlyAt(rootState)).toEqual(20);
-    expect(selectActiveSegmentIndex(rootState)).toEqual(1);
+    expect(selectCurrentlyAt(rootState)).toEqual(21);
```

Comment:
Locking should not change the layout at all. Instead of changing the test, change your implementation please.

Reply:
I have reverted the unit test.

Reply:
Thanks to #1053 I understand now that you were just fixing a test that was already failing. Good idea to submit it as a seperate PR.